### DISCUSSION TITLE
Updating error: The number of new installer packages must match the number of existing installer packages

### DISCUSSION THREAD

**Discussion by iminet** [microsoft/winget-create#268] [Q&A]
Hi All,

I'm new to winget and I published my first package 2 months ago.
I've tried to upload a fresh version of installer right now but it fails.

Command I used:
`
wingetcreate update -v 1.0.8159  -u https://github.com/iminet/DarkSwitcher/releases/download/v1.0.8159/DarkSwitcherSetup1.0.8159.exe Iminetsoft.Darkswitcher
`

Output with the error message:
``
wDPPBPuiGIoBf1tEEx0 -v 1.0.8118 Iminetsoft.Darkswitcher
Retrieving latest manifest for Iminetsoft.Darkswitcher
Downloading and parsing: https://github.com/iminet/DarkSwitcher/releases/download/v1.0.8118/DarkSwitcherSetup1.0.8118.exe...
The number of new installer packages must match the number of existing installer packages.
``

Can anybody help what's wrong with this?
Thanks in advance.

**Comment by JAMSoftwareAutomatedTasks** [comment]
We are getting the same error message when trying to update the manifest for a portable .zip file. Is there any known solution to this by now?

**Reply by mdanish-kh** [reply]
@JAMSoftwareAutomatedTasks Can you show what command you are using and what package you are talking about? I would recommend [creating an issue](https://github.com/microsoft/winget-create/issues/new/choose) for it.

**Reply by JAMSoftwareAutomatedTasks** [reply]
We tried it with a "dry run" using an updated .zip archive on a server that is not publicly accessible. The general command that we used was `wingetcreate update --urls <URL> --version <Version> <Identifier>`. 

However, we decided to implement a workaround without using winget-create as this is a faster solution compared to troubleshooting why this error message appeared. Considering it does not seem to be a widespread issue, we would rather not continue investing time into this.

Thank you for trying to help, though!
